vagrant up command fails during master step: [ERROR] Please allow iptables default FORWARD rule to ACCEPT

command run: vagrant up 
in Kubernetes folder
running with Virtualbox 6.0 on Windows 10

[...]  
  master: Installed:
    master:   iproute-tc.x86_64 0:4.16.0-1.0.5.el7     kubeadm.x86_64 0:1.12.7-1.2.3.el7
    master:   kubectl.x86_64 0:1.12.7-1.2.3.el7        kubelet.x86_64 0:1.12.7-1.2.3.el7
    master:
    master: Dependency Installed:
    master:   conntrack-tools.x86_64 0:1.4.4-5.el7_7.2
    master:   kubernetes-cni.x86_64 0:0.6.0-2.2.1.el7
    master:   kubernetes-cni-plugins.x86_64 0:0.7.5-1.0.1.el7
    master:   libnetfilter_cthelper.x86_64 0:1.0.0-10.el7_7.1
    master:   libnetfilter_cttimeout.x86_64 0:1.0.0-6.el7_7.1
    master:   libnetfilter_queue.x86_64 0:1.0.2-2.el7_2
    master:   socat.x86_64 0:1.7.3.2-2.el7
    master: Complete!
    master: net.bridge.bridge-nf-call-ip6tables = 1
    master: net.bridge.bridge-nf-call-iptables = 1
    master: Your Kubernetes VM is ready to use!
==> master: Running provisioner: shell...
    master: Running: inline script
==> master: Running provisioner: shell...
    master: Running: inline script
==> master: Running provisioner: shell...
    master: Running: inline script
==> master: Running provisioner: shell...
    master: Running: inline script
==> master: Running provisioner: shell...
    master: Running: C:/Users/tomasz/AppData/Local/Temp/vagrant-shell20200112-15956-16puilw.sh
    master: /tmp/vagrant-shell: Setup Master node -- be patient!
    master: /tmp/vagrant-shell: kubeadm-setup.sh did not complete successfully
    master: Last lines of kubeadm-setup.log:
    master: Trying to pull repository container-registry.oracle.com/kubernetes/kube-proxy ...
    master: v1.12.7: Pulling from container-registry.oracle.com/kubernetes/kube-proxy
    master: Digest: sha256:f4f9e7b70a65f4f7d751da9b97c7536b21a7ac2b301155b0685778fc83d5510f
    master: Status: Image is up to date for container-registry.oracle.com/kubernetes/kube-proxy:v1.12.7
    master: container-registry.oracle.com/kubernetes/kube-proxy:v1.12.7
    master: Checking whether docker can run container ...
    master: Checking iptables default rule ...
    master: [ERROR] Please allow iptables default FORWARD rule to ACCEPT
    master:         the way to do it:
    master:         # /sbin/iptables -P FORWARD ACCEPT
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.

--- a/Kubernetes/README.md
+++ b/Kubernetes/README.md
@@ -2,6 +2,10 @@
 This Vagrantfile will provision a Kubernetes cluster with one master and _n_
 worker nodes (2 by default).
 
+__Note:__ this Vagrantfile is deprecated in favour of the [OLCNE](../OLCNE)
+one. It can still be used to provision earlier version of Kubernetes clusters
+(e.g. 1.12).
+
 ## Prerequisites
 1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 1. Install [Vagrant](https://vagrantup.com/)

--- a/Kubernetes/scripts/provision.sh
+++ b/Kubernetes/scripts/provision.sh
@@ -81,7 +81,7 @@ done
 echo "Installing and configuring Docker Engine"
 
 # Install Docker
-yum install -y docker-engine${D4rVersion} btrfs-progs
+yum install -y docker-engine${D4rVersion} docker-cli${D4rVersion} btrfs-progs
 
 # Create and mount a BTRFS partition for docker.
 docker-storage-config -f -s btrfs -d /dev/sdb

--- a/Kubernetes/scripts/provision.sh
+++ b/Kubernetes/scripts/provision.sh
@@ -89,7 +89,7 @@ docker-storage-config -f -s btrfs -d /dev/sdb
 # Kubernetes: Docker should not touch iptables -- See Orabug 26641724/26641807
 # Alternatively you could use firewalld as described in the Kubernetes User's Guide
 # On the ol74 box, firewalld is installed but disabled by default.
-sed -i "s/^OPTIONS='\(.*\)'/OPTIONS='\1 --iptables=false'/" /etc/sysconfig/docker
+sed -i 's/^{/{\n    "iptables": false,/' /etc/docker/daemon.json
 
 # Configure insecure (non-ssl) registry if needed
 if [ -n "${Insecure}" ]

